pom.xml

```diff
@@ -20,13 +34,22 @@
         <developerConnection>scm:git:git@github.com:brooklyncentral/brooklyn-ceph.git</developerConnection>
         <tag>HEAD</tag>
     </scm>
+
     <licenses>
         <license>
             <name>Apache 2.0</name>
             <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
         </license>
     </licenses>
 
+    <!-- For the full list of developers, see contributors to https://github.com/brooklyncentral/brooklyn-ceph -->
+    <developers>
+        <developer>
+            <name>Thomas Bouron</name>
```

Comment:
Where does this appear? Should it use the actual list of developers?

Reply:
This node is a requirement for release deployment on sonatype. I added myself because that was the easiest to get going. But you're right, it should be the list of the actual devs. Can be addressed in another PR though.